portfolio/src/main/webapp/style.css

```diff
@@ -5,9 +5,44 @@
   padding: 50px;
   margin: 20px;
   position: relative;
-  left:20%;
+  left: 20%;
 }
 
-#greeting-container {
-  margin-top: 20px;
+/* Add a background to the top of the webpage */
+#topHeader {
+    width: 1000px;
+    height: 160px;
+    position: relative;
+    left: 15%;
+}
+
+/* Customize the background image */
+#topHeader img {
+    width: 1000px;
+    height: 160px;
+}
+
+/* Create and customize a navigation bar at the top of the webpage */
+.navigation {
+    width: 1000px;
+    overflow: hidden;
```

Comment:
Why 'hidden' is used here?

Reply:
Without that property, the navigation bar will be much shorter than the header background, the bar will end right after the third button, which seems a little bit wired. I want to set the background of the navigation bar the same length as the header background.